feature-libs/user/profile/components/reset-password/reset-password-component.service.ts

```diff
@@ -81,16 +80,18 @@ export class ResetPasswordComponentService {
     this.redirect();
   }
 
-  protected onError(error: Error): void {
+  protected onError(error: unknown): void {
```

Comment:
Is this considered as a breaking change?

Reply:
I would consider it a bugfix. It wasn't correct from the begging. normalized HTTP error errors weren't of this type.